experimental/Schemes/Types.jl

```diff
@@ -147,13 +147,14 @@ end
     codomain(g) === X || error("map is not compatible")
 
     if check
+      CHECK_ERROR && error("check was enabled")
       is_identity_map(compose(f, g)) && is_identity_map(compose(g, f)) || error("maps are not inverse to each other")
```

Comment:
I do not understand, why line 150 was introduced.

Reply:
It is to facilitate catching unnecessary tests in the future.

Reply:
ok. got it -- when I started reading the changes (and was interrupted many times), the discussion here had not yet started.